src/NServiceBus.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -64,5 +64,17 @@ public static TransportExtensions<RabbitMQTransport> TimeToWaitBeforeTriggeringC
             transportExtensions.GetSettings().Set(SettingsKeys.TimeToWaitBeforeTriggeringCircuitBreaker, waitTime);
             return transportExtensions;
         }
+
+        /// <summary>
+        /// Overrides the default usePublisherConfirms or value set in connection string.
+        /// </summary>
+        /// <param name="transportExtensions"></param>
+        /// <param name="publisherConfirms">Controls if publisher confirms should be used.</param>
+        /// <returns></returns>
+        public static TransportExtensions<RabbitMQTransport> UsePublisherConfirms(this TransportExtensions<RabbitMQTransport> transportExtensions, bool publisherConfirms)
```

Comment:
Nice research. If I look at the examples you mention, the ones which do not have a parameter are of the form "use X for Y" and the one with a parameter is more of the form "use Z". I think our case here is the latter, so I agree that we should keep the parameter.